velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -114,6 +114,18 @@ TEST_P(LeadLagTest, offset) {
       appendColumn(data, makeNullableFlatVector<int64_t>(expectedWindow));
 
   assertQuery(queryInfo.planNode, expected);
+
+  // Out of range offsets return default value, whereas null offsets return
+  // null.
+  queryInfo = buildWindowQuery({data}, fn("c0, c2, 99"), "order by c0", "");
```

Comment:
My understanding is that here we test with default value being a constant 99. Can we also test with null default value and a non-constant default value that has a mix of nulls and non-nulls?

Reply:
Added more tests.